src/main/java/org/opentripplanner/routing/impl/OrcaFareServiceImpl.java

```diff
@@ -137,59 +137,59 @@ public enum RideType {
         // Spaces have been removed from the route name because of inconsistencies in the WSF GTFS route dataset.
         washingtonStateFerriesFares.put(
             "Seattle-BainbridgeIsland",
-            ImmutableMap.of(Fare.FareType.regular, 9.25f, Fare.FareType.youth, 4.60f, Fare.FareType.senior, 4.60f)
+            ImmutableMap.of(Fare.FareType.regular, 9.25f, Fare.FareType.senior, 4.60f)
```

Comment:
Ferries only become free on Oct 1st, so do you want to wait until then for these changes or have an alert explaining that somewhere?

Reply:
Since this code is not in production yet, it's okay if it's not quite accurate until Oct 1st. Great catch though, impressive considering you're not in Puget Sound!

Reply:
I just searched the news about the new fare policy!